bootstrap.sh

```diff
@@ -96,7 +96,7 @@ function check_toolchains {
     exit 1
   fi
   # Check for required npm globals.
-  for util in corepack solhint; do
+  for util in corepack solhint bun; do
```

Comment:
Let's not use bun for this. We use `yarn` across the monorepo and I don't want to have to juggle tools for no reason.

Pretty confident that Charlie will agree on this

Reply:
this PR is not ready for review